pkg/operation/botanist/waiter.go

```diff
@@ -108,6 +108,37 @@ func (b *Botanist) WaitUntilEtcdMainReady(ctx context.Context) error {
 	})
 }
 
+// WaitUntilKubeAPIServerScalesDown waits until the kube-apiserver pod(s) are scaled to zero.
+// TODO: this method is a copy generalize and reuse.
```

Comment:
Is this an open TODO? It makes sense to create a util function. Plus can you then replace  [this function](https://github.com/gardener/gardener/blob/232a07d46b59689641cc65a502b0a1764b2522cd/test/integration/framework/utils.go#L327) with the new util?

Reply:
Sorry, that was meant for myself. I was supposed to do exactly that but i hide by myself under the comment of the func.